tests/PHPStan/Rules/Variables/data/bug-8113.php

```diff
@@ -34,15 +34,15 @@ function () {
 		];
 		assertType("non-empty-array<array<mixed>>&hasOffsetValue('Review', array{id: null, text: null, answer: null})&hasOffsetValue('SurveyInvitation', non-empty-array&hasOffsetValue('review', null))", $review);
 		unset($review['SurveyInvitation']['review']);
-		assertType("non-empty-array<array<mixed>>&hasOffsetValue('Review', array<mixed~'review', mixed>)&hasOffsetValue('SurveyInvitation', array<mixed~'review', mixed>)", $review);
+		assertType("non-empty-array<array<mixed>>&hasOffsetValue('Review', array{id: null, text: null, answer: null})&hasOffsetValue('SurveyInvitation', array<mixed~'review', mixed>)", $review);
```

Comment:
I think this expectation change is correct.. previously `unset($review['SurveyInvitation']['review']);` affected the offset `Review` (uppercase `R`) which was a bug IMO